test/e2e/e2e_leaderelection_test.go

```diff
@@ -141,7 +156,11 @@ func TestLeaderElection(t *testing.T) {
 			t.Logf("Only the pods in %s namespace are evicted. Pods before: %s, Pods after %s", ns2, podListBOrg, podListB)
 		}
 	} else {
-		t.Fatalf("Pods are evicted in both namespaces. For %s namespace Pods before: %s, Pods after %s. And, for %s namespace Pods before: %s, Pods after: %s", ns1, podListAOrg, podListA, ns2, podListBOrg, podListB)
+		if left && right {
+			t.Fatalf("No pods evicted. Probably none of the deschedulers were running.")
```

Comment:
should it change int t.Lof?

Reply:
The test needs to fail here as well. The test expect at least one descheduler running.

Reply:
Oh, ok, then lgtm now~~ thanks for your explain~